core/src/main/java/com/nightscout/core/model/Download.proto

```diff
@@ -0,0 +1,99 @@
+option java_package = "com.nightscout.core.model";
+
+option java_outer_classname = "G4Download";
+
+message G4Download {
+    repeated SensorGlucoseValueEntry sgv = 1; // Glucose records as reported by the G4
+    optional GlucoseUnit units = 2; // Units reported by the device
+    required string download_timestamp = 3; // ISO8601 timestamp reported by uploader
+    optional uint64 receiver_system_time_sec = 4; // Raw value of the receiver's system time
+    optional DownloadStatus download_status = 5 [default = NOT_APPLICABLE]; // Status of the download
+    optional uint32 receiver_battery = 6; // Battery level as reported by the receiver
+    optional uint32 uploader_battery = 7; // Battery level as reported by the uploader
+    repeated MeterEntry meter = 8;
+    repeated SensorEntry sensor = 9;
+    repeated CalibrationEntry cal = 10;
+}
+
+enum GlucoseUnit {
+    MGDL = 0;
+    MMOL = 1;
+}
+
+enum DownloadStatus {
+    SUCCESS = 0;
+    NO_DATA = 1;
+    DEVICE_NOT_FOUND = 2;
+    IO_ERROR = 3;
+    APPLICATION_ERROR = 4;
+    UNKNOWN = 5;
+    NOT_APPLICABLE = 6;
+}
+
+enum G4Trend {
+    TREND_NONE = 0;
+    DOUBLE_UP = 1;
+    SINGLE_UP = 2;
+    FORTY_FIVE_UP = 3;
+    FLAT = 4;
+    FORTY_FIVE_DOWN = 5;
+    SINGLE_DOWN = 6;
+    DOUBLE_DOWN = 7;
+    NOT_COMPUTABLE = 8;
+    RATE_OUT_OF_RANGE = 9;
+}
+
+enum G4Noise {
+    NOISE_NONE = 0;
+    CLEAN = 1;
+    LIGHT = 2;
+    MEDIUM = 3;
+    HEAVY = 4;
+    NOT_COMPUTED = 5;
+    MAX = 6;
+}
+
+enum ReceiverStatus {
+    RECEIVER_CONNECTED = 0;
+    RECEIVER_DISCONNECTED = 1;
+}
+
+message SensorGlucoseValueEntry {
+    required uint32 sgv_mgdl = 1; // Glucose value
+    optional uint64 sys_timestamp_sec = 2; // system timestamp of the record
+    optional uint64 disp_timestamp_sec = 3; // system timestamp of the record
```

Comment:
comment not correct. Also, explain the difference between system timestamp and display timestamp?